topic-operator/src/main/java/io/strimzi/operator/topic/K8sImpl.java

```diff
@@ -88,9 +89,18 @@ public K8sImpl(Vertx vertx, KubernetesClient client, Labels labels, String names
         vertx.executeBlocking(future -> {
             try {
                 // Delete the resource by the topic name, because neither ZK nor Kafka know the resource name
-                operation().inNamespace(namespace).withName(resourceName.toString()).delete();
-                LOGGER.debug("KafkaTopic {} deleted", resourceName.toString());
-                future.complete();
+
```

Comment:
I guess we should delete the empty space between the comment on line 91 and code on 93 to make it clear it is related?